cmd/lncli/cmd_mission_control.go

```diff
@@ -163,7 +163,7 @@ func queryMissionControl(ctx *cli.Context) error {
 var queryProbCommand = cli.Command{
 	Name:      "queryprob",
 	Category:  "Mission Control",
-	Usage:     "Estimate a success probability.",
+	Usage:     "Deprecated. Estimate a success probability.",
```

Comment:
nit: For cli commands we can add the `Hidden` param to hide it.

Reply:
Cool, done!